source/main.cpp

```diff
@@ -294,5 +330,8 @@ WUPS_CONFIG_CLOSED() {
         updateTimeThread.detach(); // Update time when settings are closed.
     }
     
+    settingsThreadActive = false;
```

Comment:
Maybe we could put `OSMemoryBarrier()` in this area somewhere to ensure that memory is synchronized between cores? I'd think after line 333 might work best.

Reply:
PPC auto-syncs the core. It might not be in between one op-code but should be fast enough for the usecase. Can't see why we should add the performace inpact of a barrier here.